server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -63,7 +64,7 @@ public void tearDown() throws Exception {
 
                 for (String aMap : map.asStrings()) {
                     try {
-                        removeMapping(args[0], args[1], aMap);
+                        removeMapping(args[0], Domain.of(args[1]), aMap);
                     } catch (IllegalArgumentException e) {
                         e.printStackTrace();
```

Comment:
Looks like bad error handling. Maybe we can introduce a commit about that?

Reply:
It's an exception in a Test teardown method.
Basically, everything else will fail just after that catch.
Furthermore this class is really bad in term of design, either we keep it that way, or we need to rewrite it completely.